Code-the-Dream-School/ffprac-team2-front
# OM_PRAC-18 Tutor search page

## One Line Description
Create a search page where user can see all tutors and be able to input teaching subjects filter available tutors.
## Requirements
Responsive layout of the tutor search page displaying all registered tutors
The page should have an input field where all users can enter teaching subjects
Upon entering the teaching subjects and submitting the form, a list of tutors who teach those subjects should be displayed.
Each Tutor card  contains information about tutor
_What this code should do_

## Notes

_Any additional info_

## Test Steps
Navigate to http://localhost:5173/tutorsearch.
Enter a teaching subject into the input field.
Submit the form and verify that a list of tutors teaching the entered subject is displayed.
_Instructions on how to test the changes_

## UI/UX Outcome
see Figma

_Attach a screenshot here_
![searchpage](https://github.com/Code-the-Dream-School/ffprac-team2-front/assets/103294778/617cebab-c025-44fc-b71c-7ebf2ac95f28)
